src/I18n/DateFormatTrait.php

```diff
@@ -135,14 +194,17 @@ public function i18nFormat($format = null, $timezone = null, $locale = null)
     {
         $time = $this;
 
+        if ($time->getTimezone()->getName() === date_default_timezone_get()) {
+            $timezone = $timezone ?: static::getDefaultOutputTimezone();
+        }
```

Comment:
Why is the default output timezone only used if the time instance's timezone matches the global default?